ruby/vampires.rb

```diff
@@ -110,3 +110,5 @@
 end
```

Comment:
For lines 70-72:

The variable type with all caps is recognized in Ruby as a constant.  Constants behave differently than local variables and should generally be avoided. For best practice stick to down_case_snake_case variable naming.